code/style.css

```diff
@@ -1,5 +1,248 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+body { 
+  background-color:gainsboro;
+  color:black;
+  font-family: Helvetica, sans-serif;
+  margin: 0;
+}
+
+h1 {
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-size: 40px;
+  line-height: 30px;
+  color:black;
+}
+
+h2 {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif
+}
+
+h3 {
+  font-size: 26px;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+h4 {
+font-size: 32px;
+font-weight: 100;
+margin-top: -16px;
+font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+margin-bottom: 16px;
+}
 
-body {
-  background-color: peachpuff;
+.sub {
+  font-family: impact, 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 24px;
+  font-weight: 500;
 }
+
+p {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-decoration: none;
+  color: black;
+}
+
+.container {
+  width: 100%;
+  flex-direction: column; 
+  max-width: 2000px;
+  text-align: center;
+}
+
+footer {
+  text-align: center;
+  text-decoration: none;
+  background-color: gainsboro;
+  color: black;
+}
```

Comment:
> I didn't get to costumize the style of the footer. I got it centered, but I do not understand, why the thing with the text decoration did not work.

You want the text styling on the anchor tag which is inside the paragraph tag, therefore you need to specify that in the CSS, like this:

```
footer p a {
  text-decoration: none;
  color: black;
}
```

meaning that it applies to all `<a`> tags inside a `<p>` tag inside the `<footer>`